.github/workflows/CI.yml

```diff
@@ -9,21 +9,14 @@ jobs:
   ci:
     runs-on: ubuntu-latest
 
-    strategy:
-      matrix:
-        node-version: [12.x, 14.x, 16.x]
-
     steps:
       - uses: actions/checkout@9bb56186c3b09b4f86b1c65136769dd318469633 # v4.1.2
 
-      - name: Use node version ${{ matrix.node-version }}
+      - name: Setup Node.js
         uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # v4.0.2
         with:
-          node-version: ${{ matrix.node-version }}
+          node-version: 22.x
```

Comment:
I saw very little value in running tests on older Node.js versions since the vast majority of what we’re running is just infrastructure. I changed this because I’m using `Set.prototype.symmetricDifference` (v22+), as well as `--expeimental-detect-module` (v20+ I think?) in order to load the otherwise-Node.js-invalid tslib.es6.js.

Reply:
The only gotcha is that using `--experimental-detect-module` means we won't know if `.es6` or whatever accidentally change to CJS, but I don't see any other way in node to load the wrong kind of file format. Other than to instead do these checks in playwright and use a real browser?